applications/doodle/src/doodle_inbound_listener_sup.erl

```diff
@@ -0,0 +1,109 @@
+%%%-------------------------------------------------------------------
+%%% @copyright (C) 2013, 2600Hz
+%%% @doc
+%%%
+%%% @end
+%%% @contributors
+%%%-------------------------------------------------------------------
+-module(doodle_inbound_listener_sup).
+
+-behaviour(supervisor).
+
+-export([start_link/0]).
+-export([init/1]).
+
+-include("doodle.hrl").
+
+-define(JSON(Proplist), {Proplist}).
```

Comment:
Please don't do this. Use `wh_json:to_proplist(JObj)` and `wh_json:from_list(Props)` for converting.